Descent3/macmain.cpp

```diff
@@ -338,7 +338,7 @@ int main(int argc, char *argv[]) {
 
   Descent = d3;
 
-  // � Initialize Metrowerks Code Profiler
+  // ¥ Initialize Metrowerks Code Profiler
```

Comment:
Is this supposed to be a Yen sign?

Reply:
Yes, it's how IDE (CodeWarrior?) on old Macs was denoted documentation in the code.

Reply:
Oh, OK. I would mark this conversation as resolved, but [apparently I can’t do that because I wasn’t the one who submitted this PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations).